api/desecapi/tests/testdomains.py

```diff
@@ -30,7 +30,6 @@ def testExpectUnauthorizedOnDelete(self):
         response = self.client.delete(url, format='json')
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
 
-
```

Comment:
pycharm highlights in light grey violations of the PEP standard (at least mine does)